Geometry/Records/interface/CaloTowerGeometryRecord.h

```diff
@@ -27,6 +28,7 @@ class CaloTowerGeometryRecord :
                 IdealGeometryRecord,
 		CaloTowerAlignmentRcd, 
 		CaloTowerAlignmentErrorRcd,
```

Comment:
@ASvyatkovskiy I agree with @ianna comment: given that we want to get rid of the old records in production, we should remove the old APE records in all the dependent ones, otherwise the job will crash as the dependent record is missing one of its constituents in the `EventSetup`. For the moment it is ok, as the new GT contains both old and new, but in the near future, when the old ones will be decommissioned, you should remove them.